public/js/deckbuilder.js

```diff
@@ -41,6 +41,13 @@ $('#addBasicsButton').click(function(e) {
   addCards(basics.Forest, $('#basicsGreen').val());
 
   renderDraft();
+
+  addCards(basics.Plains, $('#basicsWhite').val(0));
+  addCards(basics.Island, $('#basicsBlue').val(0));
+  addCards(basics.Mountain, $('#basicsRed').val(0));
+  addCards(basics.Swamp, $('#basicsBlack').val(0));
+  addCards(basics.Forest, $('#basicsGreen').val(0));
```

Comment:
I think you just want to have $('basicsGreen').val(0); and not the addCards portion - you don't need to add 0 plains to the deck.